src/server/templates/_components/form-radio-horizontal.njk

```diff
@@ -40,7 +40,7 @@
     </legend>
   {% endif %}
 
-  <div class="multiple-choice__container">
+  <div class="multiple-choice__container multiple-choise-horizontal">
```

Comment:
You can pass `inline: true` to a particular fieldset to render it inline rather than applying this class.

`multiple-choise` is also misspelt.